modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Options/Options.es.js

```diff
@@ -66,6 +67,23 @@ const Option = React.forwardRef(
 	)
 );
 
+const refreshFields = (defaultLanguageId, editingLanguageId, options) =>
+	[
+		...options.map((option) => ({
+			...option,
+			generateKeyword: isOptionValueGenerated(
+				defaultLanguageId,
+				editingLanguageId,
+				options,
+				option
+			),
+			id: random(),
+		})),
+		defaultLanguageId === editingLanguageId
+			? {...defaultOption, generateKeyword: true, id: random()}
+			: false,
+	].filter(Boolean);
```

Comment:
@marcosapmf o que seria `.filter(Boolean)`? tem alguma maneira de ser mais explicíto no que queres fazer?

por exemplo `.filter(field -> field)`?